src/applications/ivc-champva/10-10D/pages/ApplicantDependentStatus.jsx

```diff
@@ -0,0 +1,84 @@
+import ApplicantRelationshipPage, {
```

Comment:
Wrapper around ApplicantRelationshipPage, but with custom radio options. Taking this tactic because it lets us get the radio buttons + other dropdown while generating totally custom wording from current form data (not supported in radioUI).